substrait/substrait-spark/src/main/scala/io/substrait/spark/logical/ToSubstraitRel.scala

```diff
@@ -327,6 +319,27 @@ class ToSubstraitRel extends AbstractLogicalPlanVisitor with Logging {
         ))
       .build()
   }
+
+  def tree(p: LogicalPlan): String = {
+    TreePrinter.tree(visit(p))
+  }
+
+  def toProtoSubstrait(p: LogicalPlan): Array[Byte] = {
```

Comment:
This method allows for creation of a protobuf plan representation from the spark plan.